sdcflows/cli/parser.py

```diff
@@ -0,0 +1,356 @@
+may aggregate (meaning, with ``--nprocs 16 --omp-nthreads 8`` a maximum of two \
+8-CPU-threaded processes will be running at a given time). \
+Under this mode of operation, ``--nprocs`` sets the maximum number of processors \
+that can be assigned work within an *SDCFlows* job, which includes all the processors \
+used by currently running single- and multi-threaded processes. \
+If ``None``, the number of CPUs available will be automatically assigned (which may \
+not be what you want in, e.g., shared systems like a HPC cluster.""",
+    )
+    g_perfm.add_argument(
+        "--omp-nthreads",
+        "--ants-nthreads",
+        action="store",
+        type=PositiveInt,
+        help="""\
+Maximum number of threads that multi-threaded processes executed by *SDCFlows* \
+(e.g., ANTs' registration) can use. \
+If ``None``, the number of CPUs available will be automatically assigned (which may \
+not be what you want in, e.g., shared systems like a HPC cluster.""",
+    )
+    g_perfm.add_argument(
+        "--mem",
+        "--mem_gb",
+        "--mem-gb",
+        dest="memory_gb",
+        action="store",
+        type=_to_gb,
+        help="Upper bound memory limit for SDCFlows processes.",
+    )
+    g_perfm.add_argument(
+        "--debug",
+        action="store_true",
+        default=False,
+        help="Initiate in debug mode",
+    )
+    g_perfm.add_argument(
+        "--pdb",
+        dest="pdb",
+        action="store_true",
+        default=False,
+        help="Open Python debugger (pdb) on exceptions.",
+    )
+    g_perfm.add_argument(
+        "--sloppy",
+        action="store_true",
+        default=False,
+        help="Use sloppy mode for minimal footprint.",
+    )
+
+    # Control instruments
+    g_outputs = parser.add_argument_group("Instrumental options")
+    g_outputs.add_argument(
+        "-w",
+        "--work-dir",
+        action="store",
+        type=Path,
+        default=Path("work").absolute(),
+        help="Path where intermediate results should be stored.",
+    )
+    g_outputs.add_argument(
+        "-n",
+        "--dry-run",
+        action="store_true",
+        default=False,
+        help="only find estimable fieldmaps (that is, estimation is not triggered)",
+    )
+    g_outputs.add_argument(
+        "--fmapless",
+        action="store_true",
+        default=False,
+        help="Allow fieldmap-less estimation",
+    )
+    g_outputs.add_argument(
+        "--use-plugin",
+        action="store",
+        default=None,
+        type=Path,
+        help="Nipype plugin configuration file.",
+    )
+    g_outputs.add_argument(
+        "--notrack",
+        action="store_true",
+        help="Opt-out of sending tracking information of this run to the NiPreps developers. This"
+        " information helps to improve SDCFlows and provides an indicator of real world usage "
+        " crucial for obtaining funding.",
```

Comment:
Spacing is off. Adding a semantic line break.

```suggestion
        help="Opt-out of sending tracking information of this run to the NiPreps developers. "
        "This information helps to improve SDCFlows and provides an indicator of "
        "real world usage for obtaining funding.",
```